### DISCUSSION TITLE
Where do I add my BambuLab username/password?

### DISCUSSION THREAD

**Discussion by emilyboda** [greghesp/ha-bambulab#526] [Q&A] [answered]
I'm new to HA (well, a returning user, and HA looks very different now than when I first used it back in 2015). I managed to get ssh enabled, installed HACS, installed the Bambu Labs gitlab from the instructions on the readme, but I'm struggling with where to add my BambuLab username and password? I can't find any configuration settings for the integration. Can someone point me to the right place? And is this something that should be added to the readme?

**Comment by J0n-Sn0w** [comment, marked as answer]
I would also like to know. The instruction dont give any info on that.

I am dumb.. I found it. After the installation you have to go to Settings->Integrations->Add Integration-> search for Bambu Lab

**Reply by DocRonB** [reply]
Thanks for sharing!

**Reply by greghesp** [reply]
Added to the docs
https://docs.page/greghesp/ha-bambulab/setup#configuring-a-printer
